src/handlers/user-identity/tests/user-identity-handler.test.ts

```diff
@@ -6,54 +6,60 @@ import * as configuration from "../../../commons/configuration";
 import { CredentialStoreIdentityResponse } from "../../../credential-store/credential-store-identity-response";
 import { UserIdentityResponseMetadata } from "../user-identity-response-metadata";
 import { UserIdentityRequest } from "../user-identity-request";
+import * as fraudCheckService from "../../../identity-reuse/fraud-check-service";
+import { IdentityCheckCredentialJWTClass } from "@govuk-one-login/data-vocab/credentials";
+import { VerifiableCredentialJWT } from "../../../identity-reuse/verifiable-credential-jwt";
+import { getDefaultStoredIdentityHeader, sign } from "../../../../tests/acceptance/steps/utils/jwt-utils";
 
-describe("user-identity-handler tests", () => {
-  const event = () => {
-    return {
-      headers: {
-        accept: "*/*",
-        Host: "stack-name.credential-store.dev.account.gov.uk",
-        "X-Amzn-Trace-Id": "Root=1-666bf197-43c06e88748f092a5cc812a9",
-        "x-api-key": "a-pretend-api-key-value",
-        "X-Forwarded-For": "123.123.123.123",
-        "X-Forwarded-Port": "443",
-        "X-Forwarded-Proto": "https",
-        Authorization:
-          "Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiIsImtpZCI6ImVjS2lkMTIzIn0.eyJzdWIiOiJ1cm46ZmRjOmdvdi51azoyMDIyOlRFU1RfVVNFUi1TN2pjckhMR0JqLTJrZ0ItOC1jWWhWck1kbzNDVjBMbEQ3QW4iLCJleHAiOjE3NTczMjQyMTcsImlhdCI6MTc1NzMyMzkxNywiaXNzIjoiaHR0cHM6Ly9tb2NrLmNyZWRlbnRpYWwtc3RvcmUuYnVpbGQuYWNjb3VudC5nb3YudWsvb3JjaGVzdHJhdGlvbiIsImF1ZCI6Imh0dHBzOi8vY3JlZGVudGlhbC1zdG9yZS5idWlsZC5hY2NvdW50Lmdvdi51ayIsInNjb3BlIjoicHJvdmluZyJ9.Sj-2jA6mLdfkU1ryoBCNHxpBCT49o9qfqpKPMLkKwY1D6V6SvVIERGbC0X-fh8SYk2z-strc9vahvacvkrNDUQ",
-      },
-      body: JSON.stringify({
-        vtr: ["P2"],
-        govukSigninJourneyId: "govuk_signin_journey_id",
-      } satisfies UserIdentityRequest),
-    } as unknown as APIGatewayProxyEvent;
-  };
-  let newEvent: APIGatewayProxyEvent;
+enum State {
```

Comment:
Just as enums are weird would you be able to replace with:
```suggestion
const State = {
  CURRENT = "CURRENT",
  HISTORIC = "HISTORIC",
} as const;
```

Though to create a type you then need to do something like:
```typescript
type StateType = (type State)[keyof type State]
```

Happy to stick as is. If that's simpler.